src/RadhydroPulseMGint/test_radhydro_pulse_MG_int.cpp

```diff
@@ -203,21 +203,16 @@ RadSystem<MGProblem>::DefineOpacityExponentsAndLowerValues(amrex::GpuArray<doubl
 	return exponents_and_values;
 }
 
-template <> AMREX_GPU_HOST_DEVICE auto RadSystem<ExactProblem>::ComputePlanckOpacity(const double rho, const double Tgas) -> quokka::valarray<double, nGroups_>
+template <> AMREX_GPU_HOST_DEVICE auto RadSystem<ExactProblem>::ComputePlanckOpacity(const double rho, const double Tgas) -> amrex::Real
 {
 	const double sigma = scaleup * 3063.96 * std::pow(Tgas / T0, -3.5);
-	quokka::valarray<double, nGroups_> kappaPVec{};
-	kappaPVec.fillin(sigma / rho);
-	return kappaPVec;
+	return sigma / rho;
```

Comment:
I also don't understand what's going on here. I thought this function was only for gray problems.

Reply:
Same here. There are two problems in this test. ExactProblem is a single-group test with analytical planck-mean and Rossland-mean opacity and is used and the reference result. MGProblem is the multigroup problem.